src/utils/daos_hdlr.c

```diff
@@ -1895,8 +1896,8 @@ file_mkdir(struct file_dfs *file_dfs, const char *dir, mode_t *mode)
 static DIR*
 opendir_dfs(struct file_dfs *file_dfs, const char *dir)
 {
-	int rc;
-	struct fs_copy_dirent *dirp = calloc(1, sizeof(*dirp));
+	int	rc = 0;
+	struct	fs_copy_dirent *dirp = calloc(1, sizeof(*dirp));
```

Comment:
There are a couple of uses in calloc() in this code, but this is the only one that I can comment on.
```suggestion
	struct	fs_copy_dirent *dirp;
	
	 D_ALLOC_PTR(dirp);
```